hack/lib/build.sh

```diff
@@ -20,9 +20,6 @@ readonly YURT_ALL_TARGETS=(
     yurtctl
     yurt-node-servant
     yurthub
-    yurt-controller-manager
-    yurt-tunnel-server
-    yurt-tunnel-agent
```

Comment:
`release-images.sh` depends on the `build.sh`. Deleting targets here will make `release-images.sh` fail.

Reply:
Well，according to CI logs, there's no problem here.